pkg/resourceinterpreter/configurableinterpreter/configurable.go

```diff
@@ -6,14 +6,15 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/klog/v2"
 
+
```

Comment:
I understand that we don't need a new row here

Reply:
Oh sorry, I have forgotten to signoff, will so it by today itself

Reply:
Do*